user-guide/Advanced_Functionality/Databases/Indexing_database/Elasticsearch/dedicated_clustered_storage/Configuring_database/Configuring_multiple_Elasticsearch_clusters.md

```diff
@@ -21,6 +21,53 @@ Configuring multiple Elasticsearch clusters becomes crucial when on-premises set
 
 To configure this setup:
 
+## [From DataMiner 10.3.10/10.4.0 onwards](#tab/tabid-1)
+
+> [!IMPORTANT]
+> From DataMiner 10.3.10/10.4.0 onwards, configuring multiple Elasticsearch clusters should only be done via DataMiner Cube<!-- RN 36399-->.
+
+1. Go to *Apps > System Center > Database*.
+
+1. Make sure the **Type** is set to *Database per cluster*. See [Configuring the general database settings](xref:Configuring_the_database_settings_in_Cube).
+
+1. Select the checkbox next to *Multi cluster offload*.
+
+1. Specify the **File Offload Identifier**, which is the string used to identify this connection. Each connection should have a different identifier, which will be used for file offloads.
+
+1. Select *Add* in the lower right corner to add an empty Elasticsearch offload cluster to the list.
+
+   > [!NOTE]
+   > You can add an unlimited number of Elasticsearch offload clusters. The order in which the multiple clusters are listed determines their priority within the configuration. You can move offload clusters up or down with the upwards and downwards arrows next to the name of the cluster. However, the main Elasticsearch configuration always retains the highest level of priority as it is the read database.
+
+1. Specify the following database settings for each of the Elasticsearch nodes:
+
+   - **Database prefix**: The prefix that the DataMiner System will use to create the keyspaces.
+
+     > [!NOTE]
+     > The prefix has a maximum length of 20 characters. Prior to DataMiner 10.3.0 [CU5]/10.3.8<!-- RN 36503 -->, it has a maximum length of 11 characters.
+
+   - **DB server**: The IP addresses or hostnames of the nodes, separated by commas. From DataMiner 10.3.0/10.3.3 onwards, you can specify an IP address with a custom port, e.g `10.5.100.1:5555`. If no port is provided, the default Cassandra port is used instead (see [Configuring the IP network ports](xref:Configuring_the_IP_network_ports)). <!-- RN 34590 -->
```

Comment:
This will also need to be adjusted so it doesn't mention the irrelevant old version info. Also, is it correct that the default Cassandra port is used, or is this a copy/paste issue and should it in fact be the Elasticsearch port?